src/pages/Product/components/Scroller/Scroller.css

```diff
@@ -0,0 +1,14 @@
+.scroller-container img {
+    width: 279px;
+    height: 370px;
+}
+
+.scroller-container .scroller-title {
+    font-size: 16px;
+    padding: 91px 0 30px 0;
```

Comment:
жирность не указал

Reply:
<img width="609" alt="Снимок экрана 2023-09-24 в 22 21 58" src="https://github.com/DzergoS/shtor/assets/49756702/e6b5a5ee-5d10-40d6-9abe-286254846aff">

и тут отступ по макету 91 пиксель - но у тебя еще от верхнего блока добавляется отступ - что думаешь - получиться сделать в сумме 91?

Reply:
give to img w&h that's all man

Reply:
hope so

Reply:
<img width="678" alt="Снимок экрана 2023-10-02 в 21 58 35" src="https://github.com/DzergoS/shtor/assets/49756702/460e8664-0898-4eec-8baf-18dd4a1a3292">

delete all straight heights and never set them - only on icons

Reply:
<img width="904" alt="Снимок экрана 2023-10-02 в 22 00 14" src="https://github.com/DzergoS/shtor/assets/49756702/7dc00ca7-aa2f-4a7c-9399-27a25c7eaff3">
Если оставить - то оно растягивает картинки

Reply:
<img width="1258" alt="Снимок экрана 2023-10-02 в 22 01 51" src="https://github.com/DzergoS/shtor/assets/49756702/84bb20c3-5208-498f-9f00-fbebd835ec7e">
add here f.e. padding-left to make similar indets on both sides. 
+ .scroller-items and .scroller-item are one purpose divs. So choose which one to leave, and which to delete to avoid DIVOMANIA

Reply:
+